app/containers/MessageBox/index.js

```diff
@@ -818,63 +819,95 @@ class MessageBox extends Component {
 			returnKeyType: 'send'
 		} : {};
 
+		const recordAudio = showSend || !Message_AudioRecorderEnabled ? null : (
+			<RecordAudio
+				theme={theme}
+				recordingCallback={this.recordingCallback}
+				onFinish={this.finishAudioMessage}
+			/>
+		);
+
+		let safeAreaViewStyle = null;
+		let commandsPreviewAndMentions = null;
+		let replyPreview = null;
+		let content = null;
```

Comment:
It's not a good practice, since you is removing the components and turning these into variables, it'll re-render all these componentes, even them not change.